lib/stack_master/parameter_resolvers/latest_ami.rb

```diff
@@ -12,7 +12,7 @@ def resolve(value)
         owners = Array(value.fetch('owners', 'self').to_s)
         ami_finder = AmiFinder.new(@stack_definition.region)
         filters = ami_finder.build_filters_from_hash(value.fetch('filters'))
-        ami_finder.find_latest_ami(filters, owners).try(:image_id)
+        ami_finder.find_latest_ami(filters, owners)&.image_id
```

Comment:
Now that we require Ruby >= 2.4, we can use the Ruby safe navigation operator, instead of ActiveSupport's `try`.